core/src/main/java/org/elasticsearch/discovery/zen/UnicastZenPing.java

```diff
@@ -447,7 +460,7 @@ protected void sendPings(final TimeValue timeout, final PingingRound pingingRoun
         // dedup by address
         final Map<TransportAddress, DiscoveryNode> uniqueNodesByAddress =
             Stream.concat(pingingRound.getSeedNodes().stream(), nodesFromResponses.stream())
-                .collect(Collectors.toMap(DiscoveryNode::getAddress, n -> n, (n1, n2) -> n1));
+                .collect(Collectors.toMap(DiscoveryNode::getAddress, node -> node, (n1, n2) -> n1));
```

Comment:
I did but I used it wrong (as a function reference). Using it right works of course .. zzzz